analyzers/tests/SonarAnalyzer.UnitTest/Common/MetricsTest.cs

```diff
@@ -23,26 +23,6 @@ namespace SonarAnalyzer.UnitTest.Common
     [TestClass]
     public class MetricsTest
     {
-        [TestMethod]
-        public void Lines()
```

Comment:
Educational: why did we remove this test?

Reply:
It's the first commit in this PR. It removes a property `LineCount` and all it's tests. It was not used in production code :)